daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/JsonLedgerClient.scala

```diff
@@ -958,6 +962,23 @@ object JsonLedgerClient {
       }
     }
 
+    implicit val metadataReader: JsonReader[command.ContractMetadata] = json => {
```

Comment:
I’d probably just drop this, if not, let’s at least add a TODO to double check this & add tests once the JSON API exposes this.

Reply:
Added a ticket to double-check, once the JSON API is ready.